pulser/register.py

```diff
@@ -63,6 +63,7 @@ def from_coordinates(
         coords: np.ndarray,
         center: bool = True,
         prefix: Optional[str] = None,
+        labels: Optional[ArrayLike] = None,
```

Comment:
I think instead of `ArrayLike`, we can be a bit stricter and make it a `collections.abc.Sequence[QubitId]` (I know, it's ugly but it's the way we got not to confuse it with `pulser.Sequence`)

Reply:
I'll try that